openapi/OpenAPI/Program.cs

```diff
@@ -0,0 +1,20 @@
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddOpenApi();
+
+var app = builder.Build();
+
+if (app.Environment.IsDevelopment())
+{
+    app.MapOpenApi();
+    app.UseSwaggerUI(options =>
+    {
+        options.SwaggerEndpoint("/openapi/v1.json", "Api");
+    });
+}
+
+app.UseHttpsRedirection();
+
+app.MapGet("/", () => "Hello World!");
```

Comment:
can we have a customized schema here? we apply many filters and customizations over generated schema file. let's dig more in here so that we'll migrate smoothly in baked.

Reply:
If you are talking about the area where we enter the token in the UI, it seems that we cannot do it. Because we determine it with SwaggerGen. It seems like we can't do it from SwaggerUI. So we need full swashbuckle. If you are asking if there is configuration for OpenAPI, yes
```csharp
builder.Services.AddAuthentication().AddJwtBearer();
builder.Services.AddAuthorization(o =>
{
    o.AddPolicy(“ApiTesterPolicy”, b => b.RequireRole(“tester”));
});
builder.Services.AddOpenApi();

var app = builder.Build();

app.MapOpenApi()
    .RequireAuthorization(“ApiTesterPolicy”);
```
schema can be updated in this way but it is not reflected in the ui.

Reply:
hmm, can you check what schema manipulations we're doing in baked to see if we can achieve them using this new open api feature. if not, we'll leave it as researched not used